auth/routes/v1.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response, Router } from "express";
+import {
+  createUser,
+  deleteUser,
+  getUser,
+  loginUser,
+  updateUser
+} from "../controllers/userController";
+
+export default function (app: Router, passport: any) {
+  /********  Api details  **********/
+  app.get("/api", function (req: Request, res: Response) {
+    res.json({ message: "Internado", version: "v1.0.0" });
+  });
+  /********************************/
+
+  // Login user
+  app.post("/api/user/login", loginUser);
+  app.options("/api/user/login", corsAllowMethods("POST"));
```

Comment:
Could we make these `/user` directly? To make it more consistent with the search api which uses `/search` directly.